README.md

```diff
@@ -120,21 +120,21 @@ You have two options to install the verifier.
 #### Option 1: Install via go
 
 ```
-$ go install github.com/slsa-framework/slsa-verifier/cli/slsa-verifier@v1.3.2
+$ go install github.com/slsa-framework/slsa-verifier/v2/cli/slsa-verifier@v2.0.0
```

Comment:
Oh, sorry. I found this doesn't work. 🙇‍♂️ 
To install slsa-verifier v2 by go install, we have to release v2.0.1 or later.

```console
go install github.com/slsa-framework/slsa-verifier/v2/cli/slsa-verifier@v2.0.0
go: github.com/slsa-framework/slsa-verifier/v2/cli/slsa-verifier@v2.0.0: github.com/slsa-framework/slsa-verifier@v2.0.0: invalid version: module contains a go.mod file, so module path must match major version ("github.com/slsa-framework/slsa-verifier/v2")
```

I'll send a pull request to fix the document.

Reply:
Created the pr.

- https://github.com/slsa-framework/slsa-verifier/pull/376